yarn.lock

```diff
@@ -2756,7 +2756,7 @@ address@^1.0.1:
   resolved "https://registry.npmjs.org/address/-/address-1.1.2.tgz"
   integrity sha512-aT6camzM4xEA54YVJYSqxz1kv4IHnQZRtThJJHhUMRExaU5spC7jX5ugSwTaTgJliIgs4VhZOk7htClvQ/LmRA==
 
-agent-base@6, agent-base@^6.0.0:
```

Comment:
Is it necessary to commit `yarn.lock` changes? 🙂

Reply:
I've deleted yarn lock locally and reinstalled with the most recent master merged in.  Not sure why these changes are here nor why they would be flagged as new, as nothing has been added on this branch.

Reply:
You can remove the yarn.lock changes by doing `git reset --soft HEAD~1` and then re-committing all the files except the yarn.lock 🙂 Assuming that the yarn.lock was updated in the most recent commit on your branch actually

Reply:
I was able to re-source yarn lock off the master branch and that seemed to do the trick. `git checkout origin/master -- yarn.lock`